src/WebformBuilder.js

```diff
@@ -1117,8 +1117,12 @@ export default class WebformBuilder extends Component {
         }
       }
       else {
-        this.preview._data[changed.instance._data.key] = changed.value;
-        this.webform._data[changed.instance._data.key] = changed.value;
+        const dataPath = _.get(defaultValueComponent, 'component.type') === 'datagrid'
```

Comment:
Does this also not affect EditGrid? Seems strange to have a specific case for just datagrid here.

Reply:
No, because we don't have default value settings for the EditGrid